client/src/components/attendance/AttendanceForm.js

```diff
@@ -276,12 +277,27 @@ export default function AttendanceForm(props) {
         renderCell: (params) => (
           <Slider className={classes.slider} onChangeCommitted={(e, value) => handleCapacityChange(value, params)}
                   disabled={params.getValue('isAbsent')}
-                  step={1} min={1} max={4} defaultValue={params.getValue('capacity')} data-capacity={params.getValue('capacity')} />
+                  step={1} min={1} max={4} defaultValue={params.getValue('capacity')}
+                  data-capacity={params.getValue('capacity')}/>
         )
       }
     )
   }
 
+  let history = useHistory()
+
+  const handleSeeAll = () => {
+    history.push("/events")
+  }
+
+  /**
+   * Checks if there are any Not Submitted events, and displays a section header for the dropdown list, if there are.
+   */
+
```

Comment:
Change to single inline comment –  block comments should usually be reserved for top level code blocks or important big sections.